mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -39,6 +39,10 @@ class DiodeRun_V1(object):
     # ### VACUUM ###
     # Cathode anode distance
     D_CA = 1e-3
+    # Vacuum bias of the highest voltage grid relative to the cathode
+    V_GRID = None
```

Comment:
This can be removed after implementing the changes discussed below. Main thing is that a diode cannot have a grid - then it would at minimum be a triode.